receptor-python-worker/receptor_python_worker/work.py

```diff
@@ -5,7 +5,7 @@
 import queue
 from pathlib import Path
 import json
-import pkg_resources
+import importlib.metadata
```

Comment:
Changing this is not enough. There's code in this file that still calls `pkg_resources` — it will fail with `NameError` until fixed to make new calls.